cookbooks/active_dau.md

```diff
@@ -0,0 +1,30 @@
+# Active DAU
+
+An "Active User" is defined as a client who has total_daily_uri >= 5 URI for a given date. 
```

Comment:
Please mark total_daily_uri as code e.g. `total_daily_uri`. This will prevent any issues with the spelling CI.